azure-mvn-settings.xml

```diff
@@ -1,12 +1,19 @@
 <settings>
   <profiles>
     <profile>
-      <id>google-mirror-jboss-proxy</id>
-      <repositories>
+      <id>azure-jboss-proxy</id>
+        <repositories>
+          <repository>
+          <id>dev-azure-com-quarkus-ci-quarkusmavenfeed</id>
+          <url>https://pkgs.dev.azure.com/quarkus-ci/_packaging/QuarkusMavenFeed/maven/v1</url>
```

Comment:
I can't access the repo:
````
{"$id":"1","innerException":null,"message":"The user does not have a license for the extension ms.feed.","typeName":"Microsoft.VisualStudio.Services.ExtensionManagement.Sdk.Server.ExtensionNotLicensedException, Microsoft.VisualStudio.Services.ExtensionManagement.Sdk.Server","typeKey":"ExtensionNotLicensedException","errorCode":0,"eventId":3000}
```

Also:

* if it a maven central mirror, if so what's the latency
* does it provide a mirror for "oss.snapshot"
* how does it work for publication?